### ISSUE TITLE
admin user view

### ISSUE THREAD

**Issue by natehill** [esmero/archipelago-deployment#7]
It looks like logging in as admin user makes it impossible to see the site as a regular user, you are only exposed to admin tools. Maybe Drupal has some 'switch user' functionality that can help with this. If yes, this would be good to add to [documentation](https://github.com/esmero/archipelago-doucmentation)

**Comment by DiegoPino** [comment]
@natehill yes, there is a user Switch block that can be enabled for admin users. We also ship with 2 other users so people can see that and eventually we will be building a theme for admins that looks a bit closer to the one normal users have so we configure have same blocks, same normal displays for admin users. Could you please make this ISSUE title a bit more explicit? Like "Allow Admin user to see site as normal user" or something in that tone. Having titles that explain what is being requested helps keep track of the needs when looking at the full ISSUE list.
